### ISSUE TITLE
"Indent" rule throws false positives for object under ternary.

### ISSUE THREAD

**Issue by phyllisstein** [eslint/typescript-eslint-parser#452]
### What version of TypeScript are you using?
`v2.8.0-rc`

### What version of `typescript-eslint-parser` are you using?
`v14.0.0`

### What code were you trying to parse?
```ts
const firebaseApp = firebase.apps.length
  ? firebase.app()
  : firebase.initializeApp({
      apiKey: __FIREBASE_API_KEY__,
      authDomain: __FIREBASE_AUTH_DOMAIN__,
      databaseURL: __FIREBASE_DATABASE_URL__,
      projectId: __FIREBASE_PROJECT_ID__,
      storageBucket: __FIREBASE_STORAGE_BUCKET__,
      messagingSenderId: __FIREBASE_MESSAGING_SENDER_ID__,
    })
```

### What did you expect to happen?
ESLint should not flag the object as incorrectly indented (cf. [results from the standard ESLint parser](https://eslint.org/demo/#eyJ0ZXh0IjoiY29uc3QgZmlyZWJhc2UgPSByZXF1aXJlKCdmaXJlYmFzZScpXG5cbmNvbnN0IGZpcmViYXNlQXBwID0gZmlyZWJhc2UuYXBwcy5sZW5ndGhcbiAgXHQ/IGZpcmViYXNlLmFwcCgpXG4gIFx0OiBmaXJlYmFzZS5pbml0aWFsaXplQXBwKHtcbiAgICBcdGFwaUtleTogJ19fRklSRUJBU0VfQVBJX0tFWV9fJyxcbiAgICAgIFx0YXV0aERvbWFpbjogJ19fRklSRUJBU0VfQVVUSF9ET01BSU5fXycsXG4gICAgICBcdGRhdGFiYXNlVVJMOiAnX19GSVJFQkFTRV9EQVRBQkFTRV9VUkxfXycsXG4gICAgICBcdHByb2plY3RJZDogJ19fRklSRUJBU0VfUFJPSkVDVF9JRF9fJyxcbiAgICAgIFx0c3RvcmFnZUJ1Y2tldDogJ19fRklSRUJBU0VfU1RPUkFHRV9CVUNLRVRfXycsXG4gICAgICBcdG1lc3NhZ2luZ1NlbmRlcklkOiAnX19GSVJFQkFTRV9NRVNTQUdJTkdfU0VOREVSX0lEX18nLFxuICAgIH0pIiwib3B0aW9ucyI6eyJwYXJzZXJPcHRpb25zIjp7ImVjbWFWZXJzaW9uIjo4LCJzb3VyY2VUeXBlIjoibW9kdWxlIiwiZWNtYUZlYXR1cmVzIjp7fX0sInJ1bGVzIjp7ImluZGVudCI6Mn0sImVudiI6eyJub2RlIjp0cnVlLCJicm93c2VyIjp0cnVlfX19)).

### What happened?
ESLint w/ `typescript-eslint-parser` flagged the object as incorrectly indented:

```
warning: Expected indentation of 6 spaces but found 8 (indent) at src/app/flux/lib/configure-store.ts:38:1:
  36 |     ? firebase.app()
  37 |     : firebase.initializeApp({
> 38 |         apiKey: __FIREBASE_API_KEY__,
     | ^
  39 |         authDomain: __FIREBASE_AUTH_DOMAIN__,
  40 |         databaseURL: __FIREBASE_DATABASE_URL__,
  41 |         projectId: __FIREBASE_PROJECT_ID__,
```

**Comment by phyllisstein** [comment]
👋 Any chance one of the maintainers could take a look at this one?

**Comment by armano2** [comment]
issue is no longer present in v21.0.1

**Comment by phyllisstein** [comment]
💖Thanks for the update! I'll go ahead and close this guy.
